### DISCUSSION TITLE
How to set up a background image

### DISCUSSION THREAD

**Discussion by JATIN-RAGHAV** [catppuccin/nvim#795] [Q&A] [answered]
How can I set up a backgroun image. 
Either I can make the background transparent and make the default terminal backgroun image appear. That works.
But how can I use an image as background for my nvim instead of making the bg transparent, which will appear even when I have opened a floating gui like Lazy??

**Comment by vollowx** [comment, marked as answer]
`use an image as background for my nvim instead of making the bg transparent` is simply impossible only through Neovim, nor the matter of `catppuccin/nvim`, please check the document of the terminal you use.
